anago

```diff
@@ -554,13 +557,20 @@ announce () {
   local announcement_text=/tmp/$PROG-announce.$$
 
   ((FLAGS_nomock)) || mailto=$USER@$DOMAIN_NAME
+  if [[ -n "${FLAGS_mailto}" ]]; then
+    mailto="${FLAGS_mailto}"
+  fi
 
   if [[ -n "$PARENT_BRANCH" ]]; then
     create_branch_announcement
   else
     create_announcement
   fi > $announcement_text
 
+  ((FLAGS_yes)) \
+   || common::askyorn -e "Pausing here. Confirm announce to $mailto" \
```

Comment:
I had intended that the confirmation would appear regardless of whether `--mailto` had been used to override. Even in the normal workflow, I find it useful to have control over when the email gets sent. Do you prefer if I change it to only prompt if `mailto` was customized?